ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1,2 @@
-m_EditorVersion: 2018.4.16f1
```

Comment:
You're changing the unity version here, don't know if this is smart to upload to git